plugin/src/integrationTest/kotlin/com/vanniktech/maven/publish/Subjects.kt

```diff
@@ -176,3 +182,36 @@ class SourcesJarSubject private constructor(
     }
   }
 }
+
+class PomSubject private constructor(
+  failureMetadata: FailureMetadata,
+  private val artifact: Path,
+  private val result: ProjectResult,
+) : ArtifactSubject(failureMetadata, artifact) {
+
+  companion object {
+    private val BUILD_RESULT_SUBJECT_FACTORY: Factory<PomSubject, Pair<Path, ProjectResult>> =
+      Factory { metadata, actual -> PomSubject(metadata, actual.first, actual.second) }
+
+    @JvmStatic
```

Comment:
Do we need this? I thought we are Kotlin only

Reply:
We don't just came from copying the default java sample

Reply:
[67a10ea](https://github.com/vanniktech/gradle-maven-publish-plugin/pull/461/commits/67a10ea8b42bf19c191931743e8ddbe48ef5bf4c)